pkg/redact/redact.go

```diff
@@ -35,15 +35,15 @@ type Redactor interface {
 
 // Redactions are indexed both by the file affected and by the name of the redactor
 type RedactionList struct {
-	ByRedactor map[string][]Redaction
-	ByFile     map[string][]Redaction
+	ByRedactor map[string][]Redaction `json:"byRedactor" yaml:"byRedactor"`
+	ByFile     map[string][]Redaction `json:"byFile" yaml:"byFile"`
```

Comment:
@CodeLingoBot capture JSON tags should have corresponding YAML tags
When adding JSON tags to struct fields, also add corresponding YAML tags for consistent marshalling